### ISSUE TITLE
[Fabric] Version 2.0.1 is not compatible with minecraft version 1.20.1

### ISSUE THREAD

**Issue by techheadfred-git** [tebexio/Tebex-Minecraft#23]
Crash on loading.
- Server type and version Fabric 1 20.1
- Tebex plugin version 2.0.1
- Description of the issue with any error messages, stack traces, or relevant logs.
- See crash log
[crash-2024-01-07_14.21.43-server.txt](https://github.com/tebexio/Tebex-Minecraft/files/14174774/crash-2024-01-07_14.21.43-server.txt)

**Comment by WildBamaBoy** [comment]
Thanks for the report. It appears this is caused by **StyledPlayerList**, not Tebex:

```
java.lang.NullPointerException: Cannot invoke "eu.pb4.styledplayerlist.config.Config.isPlayerHidden(net.minecraft.class_3222)" because the return value of "eu.pb4.styledplayerlist.config.ConfigManager.getConfig()" is null
```

**Comment by techheadfred-git** [comment]
StyledPlayerList is not caching the server, it works fine without the Tebex
mod.
I will remove StyledPlayerList and re-test. If it crashes as expected, I
will submit a new crash report.

**Comment by techheadfred-git** [comment]
Please reopen, as this was _definitely_ Tebex 2.0.1, please see the following:

[02:18:24] [main/ERROR]: Incompatible mods found!
net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
A potential solution has been determined, this may resolve your problem:
	 - Replace mod 'Tebex' (tebex) 2.0.1 with any version that is compatible with:
		 - minecraft 1.20.1
**_More details:
	 - Mod 'Tebex' (tebex) 2.0.1 requires version 1.16.5 of 'Minecraft' (minecraft), but only the wrong version is present: 1.20.1!_**
	 
	 I note Tebex 2.0.2 is now available, I will retest with 2.0.2
